hack/cmd_util.sh

```diff
@@ -182,16 +184,18 @@ function os::cmd::internal::expect_exit_code_run_grep() {
 	done
 
 	if (( cmd_succeeded && test_succeeded )); then
-		os::text::print_green "SUCCESS after ${time_elapsed}s: ${name}"
+		os::text::print_green "SUCCESS after ${time_elapsed}s: ${name}" | tee "{JUNIT_REPORT_OUTPUT}"
```

Comment:
> Make this command work as expected

Without that variable set right now:
 * no file is written
 * no error is raised
 * nothing fails to run

What else is expected?